scripts/test.py

```diff
@@ -0,0 +1,8 @@
+import requests
```

Comment:
give a more specific name instead of test.py

Reply:
Sure, this one is just to test the github workflows...When it is ready for review, will change its name and add more comments